code/game/objects/items/latexballoon.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/latexballon
 	name = "latex glove"
-	desc = "" //todo
+	desc = "Sterile and airtight."

```

Comment:
Seriously though who todo's a FREAKING DESCRIPTION in the first place. JUST TYPE IN LITERALLY ANYTHING IT TAKES 5 SECONDS.